assets/elm/Target/Helper.elm

```diff
@@ -24,16 +24,16 @@ abbreviation =
             , ( 179, "OTT" )
             ]
     in
-        Dict.fromList abbr
+    Dict.fromList abbr
 
 
 getDataSources : String -> Cmd Msg
 getDataSources url =
     let
         datasourceUrl =
-            (Debug.log "url" url) ++ "/data_sources.json"
+            Debug.log "url" url ++ "/data_sources.json"
     in
-        Http.send AllDataSources (Http.get datasourceUrl dataSourceDecoder)
+    Http.send AllDataSources (Http.get datasourceUrl dataSourceDecoder)
```

Comment:
same here re: `elm-format`... I won't comment further about these, but it looks like it's across much of this file